### ISSUE TITLE
Lower body tracking(IK) not there 

### ISSUE THREAD

**Issue by SabinMG** [oculus-samples/Unity-Movement#5]
Is fullbody tracking supported  with Movement SDK? 

I run the sample scene "MovementRetargeting", but only upper body movements are tracked. Lower part always move along with
upper part. Seems like no IK behaviours on leg. 

Nothing much explained in the documentation as well, just body tracking.

Could you please confirm this?.

Regards
Sabin M. George

**Comment by dilmerv** [comment]
That's correct Sabin, the lower part of the body is currently not supported by the Movement SDK.
